app/models/work-item.ts

```diff
@@ -167,6 +167,31 @@ export async function updateWorkItemStatus(
   }
 }
 
+/**
+ * Update the status of work items by job ID.
+ * @param tx - the transaction to use for the update
+ * @param jobID - the jobID associated with the work items
+ * @param oldStatuses - restricts the updates to work items where the status is in oldStatuses
+ * @param newStatus - the value of the updated status
+ */
+export async function updateWorkItemsStatusByJobId(
```

Comment:
Related to prior message, I could see this being a method on the job, i.e. `job.updateWorkItemStatuses`